docs/java-rest/high-level/search/point-in-time.asciidoc

```diff
@@ -53,19 +53,8 @@ include-tagged::{doc-tests}/SearchDocumentationIT.java[search-point-in-time]
 A search request with a point in time does not accept these parameters:
 `indices`, `indicesOptions` `routing`, `preference`, and `ccsMinimizeRoundtrips`.
 
-==== Paginate search results with point in time
```

Comment:
What's the reason we remove pagination with search after from documentation?  Is it because it is obvious enough?

I am not familiar when SearchDocumentationIT.java is executed, if we remove this part from docs, should we also remove `testSearchAfterWithPointInTime()` from `SearchDocumentationIT.java`?

Reply:
>  should we also remove testSearchAfterWithPointInTime() from SearchDocumentationIT.java

++. I removed it in https://github.com/elastic/elasticsearch/pull/73874/commits/014cea2f6c19a6a018433e3eef509374bce71488.

I think the explanation is good enough.